backend/src/openarchiefbeheer/zaken/tasks.py

```diff
@@ -21,14 +21,24 @@ def retrieve_and_cache_zaken_from_openzaak() -> None:
     zrc_service = Service.objects.get(api_type=APITypes.zrc)
     zrc_client = build_client(zrc_service)
 
+    today = datetime.date.today()
+    query_params = {
+        "expand": "resultaat,resultaat.resultaattype,zaaktype,rollen",
+        "archiefnominatie": "vernietigen",
+        "einddatum__isnull": False,
+        "einddatum__lt": today.isoformat(),
+    }
+    if zaken_in_db := Zaak.objects.exists():
+        yesterday = today - datetime.timedelta(
+            days=2
+        )  # Since we can only use gt and not gte
+        query_params.update({"einddatum__gt": yesterday.isoformat()})
```

Comment:
Is this correct? What happens if indexing has failed for a X amount of days, will re-indexing then get the missing cases or will they remain unindexed? I would expect to check against the latest imported zaak already in the DB? Or am I misinterpreting this code?

Reply:
Ahh that's a good point :thinking: I will change it to use the latest eindatum in the database